cli/cmd/resources.go

```diff
@@ -57,7 +57,7 @@ var (
 
 	envTokensClient = resourcemanager.NewClient(
 		httpClient, cliLogger,
-		"token", "environmenttokens",
+		"environmenttoken", "environmenttokens",
```

Comment:
do we want this change? same for `organizationinvite`?

Reply:
So, we need to either change the cloud BE to use the same nomenclature of the CLI or keep it like this, the problem is that the registry won't find the client when the definition is `type: OrganizationInvite` or `type: EnvironmentToken` as it is registered as `token` and `invite`.

We might need to change that tho

Reply:
I think for the environment tokens we need to do this, but for invites, it should work!

Reply:
🤔 Got it! I would prefer to match the cmd with the resource type (`EnvironmentToken` and `Invite`). I'm not sure if changing the cloud BE at point makes sense.